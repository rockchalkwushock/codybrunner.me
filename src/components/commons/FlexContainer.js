import styled from 'styled-components'

const FlexContainer = styled.div`
  align-items: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
`

FlexContainer.displayName = 'FlexContainer'

export default FlexContainer
