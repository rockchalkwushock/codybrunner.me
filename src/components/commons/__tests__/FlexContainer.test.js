import React from 'react'
import { shallow } from 'enzyme'

import FlexContainer from '../FlexContainer'

describe('Component: <FlexContainer />', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<FlexContainer />)
  })

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
  test('should have styles', () => {
    expect(wrapper).toHaveStyleRule('align-items', 'center')
    expect(wrapper).toHaveStyleRule('display', 'flex')
    expect(wrapper).toHaveStyleRule('flex-direction', 'column')
    expect(wrapper).toHaveStyleRule('justify-content', 'center')
  })
})
