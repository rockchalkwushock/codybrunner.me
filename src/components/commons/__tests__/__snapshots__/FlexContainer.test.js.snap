// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <FlexContainer /> should render correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
/>
`;
